README.md

```diff
@@ -1,75 +1,168 @@
+
+#### What next?
+
+Now that your analysis is complete, there are a couple things you might want to do after an initial integration:
+
+- **Double-check your results.** Some analysis methods may produce partial or unexpected results depending on what information was available when you ran the analysis. If something seems wrong, [our debugging guide](./docs/walkthroughs/debugging-your-integration.md) can help you diagnose and debug your integration.
+
+- **Scan for issues and generate a compliance report.** Once your analysis is ready, we'll automatically queue an issue scan and report the results in the web application. If you want to try out different issue scanning configurations, you can [change your policy](TODO) or [run a manual issue scan](TODO). Once an issue scan is complete, you can also [generate a report](TODO) from the web application.
+
+- **Set up FOSSA in your CI.** You can also use your issue scan results as inputs to CI scripts. For GitHub repositories, you can use FOSSA's [native GitHub integration](TODO) to report a status check on your PRs. For other CI integrations, you can also [use `fossa test`](TODO) to get programmatic issue status in CI.
+
+### Using Spectrometer as a standalone tool
+
+You can also use Spectrometer as a standalone tool for dependency analysis. Note that some features (like vendored dependency identification) won't work without a full FOSSA account and integration, since they rely on backend services.
+
+You can get the output of `fossa` using the `--output` flag:
+
+```sh
+fossa analyze --output
+```
+
+Note that Spectrometer's `--output` format is **not considered stable**. While we will try our best to maintain backwards compatibility, this output format may change without warning between releases.
+
+## User Manual
+
+For most users, Spectrometer should work out-of-the-box without any configuration. Just get an API key, run `fossa analyze`, and view your results in the FOSSA web application.
+
+Users who need more advanced customizations or features should see the [User Manual](./docs/README.md). Some common topics of interest include:
+
+- [Debugging your integration](./docs/walkthroughs/debugging-your-integration.md)
+- [Excluding developer tools from your dependencies](./docs/walkthroughs/excluding-test-dev-deps.md)
+- [Specifying vendored dependencies](./docs/walkthroughs/specifying-vendored-deps.md)
+- [Adding manual dependencies](./docs/walkthroughs/integrating-with-custom-tools.md)
 
 ## Reporting Issues
 
-If you are experiencing an issue related to the results on the FOSSA
-website/dashboard, please contact [support@fossa.com](mailto:support@fossa.com)
+If you've found a bug or need support, the best way to get support is to email [support@fossa.com](mailto:support@fossa.com).
+
+Make sure to include reproduction steps and any relevant project files (e.g. `pom.xml`s, `package.json`s, etc.).
```

Comment:
Do we want to also suggest they preemptively provide a debug bundle or at least a `--debug` log?